apk-generator/v2/app/generator/generator.py

```diff
@@ -196,8 +205,22 @@ def download_event_data(self):
         self.save_file_in_temp_assets('speakers')
         self.save_file_in_temp_assets('sponsors')
         self.save_file_in_temp_assets('tracks')
+	self.save_file_in_temp_images_speakers('images/speakers')
         logger.info('Download complete')
 
+    def save_file_in_temp_images_speakers(self, end_point='event'):
+        """
+        Save response from specified end_point in drawable directory
+        :param end_point:
+        :return:
+        """
+        if self.api_link:
+            response = requests.get(self.api_link + '/' + end_point)
```

Comment:
This won't work. What you are doing is making api call which is not even exists.
that is `https://eventyay.com/api/v1/events/6/images/speakers`